### DISCUSSION TITLE
How to make simultaneous press work

### DISCUSSION THREAD

**Discussion by potul** [alf45tar/PedalinoMini#499] [Q&A]
I've been trying to get simultaneous press to work without success.
In addition to create a control with 2 pedals in it, and create the right actions... is there anything else I need to configure? I can't make the simultaneous press to trigger any action, and the actions of the individual buttons are triggering, so the whole logic doesn't seem to work.

**Comment by alf45tar** [comment]
3.2.2 has a bug fixed in 3.2.3.

**Reply by potul** [reply]
Ok. I will have to wait for the release then. Is 3.2.1 having the same issue?

3.2.3 is only available for sponsors, right?

**Reply by alf45tar** [reply]
I do not know exactly when the bug has been added. You have to try old releases.
Yes, 3.2.3 is for sponsor only.
